api/client/commands.go

```diff
@@ -737,16 +737,19 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 
 	if *attach || *openStdin {
 		if cmd.NArg() > 1 {
+			close(hijacked)
```

Comment:
maybe add `defer close(hijacked)` below `hijacked := make(chan io.Closer)` is a better solution